chsdi/templates/htmlpopup/windatlas50.mako

```diff
@@ -0,0 +1,738 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+  <%
+    baseUrl = request.registry.settings['api_url']
+  %>
+  <iframe src="${baseUrl}/rest/services/all/MapServer/${c['layerBodId']}/${c['id']}/extendedHtmlPopup?lang=${lang}" width="100%" height="400" frameborder="0"  style="border: 0"></iframe>
+</%def>
+
+<%def name="extended_info(c, lang)">
+
+<%
+  from chsdi.lib.raster.georaster import get_raster
+
+  r = get_raster('COMB')
```

Comment:
@gjn shall I re-sync all profile stuff? Currently only the indexes are there:

du -hs /var/local/profile/swissalti3d/
508K	/var/local/profile/swissalti3d/